### ISSUE TITLE
argocd having diff issues with the list 

### ISSUE THREAD

**Issue by rmechi** [wiz-sec/charts#303]
![image](https://github.com/wiz-sec/charts/assets/92652774/072f6d81-e9d3-4152-8484-bfc5040eb67b)

we are trying to install wiz using helm chart using ArgoCD.

https://github.com/wiz-sec/charts/blob/3a8421a515772a58afbda025c824941f344c4cbb/wiz-sensor/templates/openshift.yaml#L42

is this needs to be. sorted ?

**Comment by ariknem** [comment]
are you installing on openshift?
this is pretty weird anyway..

is this the only diff it's reporting?

**Comment by clutchbeyers** [comment]
> are you installing on openshift? this is pretty weird anyway..
> 
> is this the only diff it's reporting?

Yes this is on RedHat Openshift on AWS (ROSA) by way of argo/helm.

**Comment by clutchbeyers** [comment]
Not that this should have any relation since its different resources, we do setup an anyuid role and rolebinding to allow for the pod security contexts to work for our automation across many clusters.

`kind: Role
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: anyuid-scc
  namespace: wiz
rules:
  - verbs:
      - use
    apiGroups:
      - security.openshift.io
    resources:
      - securitycontextconstraints
    resourceNames:
      - anyuid
---
kind: RoleBinding
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: anyuid-scc-wiz
  namespace: wiz
subjects:
  - kind: Group
    name: system:serviceaccounts:wiz
    apiGroup: rbac.authorization.k8s.io
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: Role
  name: anyuid-scc`

**Comment by ariknem** [comment]
You need this in addition to the SCC that we put in the helm chart?
Shouldn't this line in openshift.yaml take care of this:

runAsUser:
  type: RunAsAny

**Comment by ariknem** [comment]
Anyway we will sort these values alphabetically. New chart will probably be out today.

**Comment by ariknem** [comment]
This is now fixed with version 1.0.3891 of the chart

**Comment by ariknem** [comment]
ah yes. the charts are shared for the sensor, kubernetes-connector and admission-controller. but the sensor doesn't need this due to the SCC. I believe only the other two need this. I'll pass on your comment to the appropriate people in Wiz :)

**Comment by clutchbeyers** [comment]
So one more question on this 1.0.3891 of the sensor chart. Does this automatically get pulled in via 0.1.92 of the wiz-sec/wiz-kubernetes-integration chart dependencies? Or will I need to set a values override to get this? Right now we are using the latest 0.1.92 and it still shows helm.sh/chart=wiz-sensor-1.0.3828

**Comment by clutchbeyers** [comment]
https://github.com/wiz-sec/charts/blob/master/wiz-kubernetes-integration/Chart.yaml

**Comment by ariknem** [comment]
i think "helm dependency update" is required to update the local cache

**Comment by ariknem** [comment]
you were right! we have some kind of race condition in our publish flow (we will fix it don't worry). for now published a new version manually and it looks fine now:
- name: wiz-sensor
  repository: https://wiz-sec.github.io/charts
  version: 1.0.3891
